src/Solnet.Rpc/IRpcClient.cs

```diff
@@ -810,11 +817,20 @@ public interface IRpcClient
         /// <returns>Returns an object that wraps the result along with possible errors with the request.</returns>
         RequestResult<TransactionMetaSlotInfo> GetTransaction(string signature, Commitment commitment = Commitment.Finalized);
 
+        /// <summary>
+        /// Returns transaction details for a confirmed transaction.
+        /// </summary>
+        /// <param name="signature">Transaction signature as base-58 encoded string.</param>
+        /// <returns>Returns an object that wraps the result along with possible errors with the request.</returns>
+        [Obsolete]
+        RequestResult<TransactionMetaSlotInfo> GetConfirmedTransaction(string signature);
+        
         /// <summary>
         /// Gets the total transaction count of the ledger.
         /// </summary>
         /// <param name="commitment">The state commitment to consider when querying the ledger state.</param>
         /// <returns>Returns a task that holds the asynchronous operation result and state.</returns>
+        [Obsolete]

```

Comment:
Looks like you marked `GetTransactionCountAsync` as Obsolete when it isn't? At least from what I'm seeing in the docs it's not

Reply:
you are right

Reply:
fixed

Reply:
Looks like it's still there, you might have forgotten to add the changes to that file in the most recent commit, just fix that and I'll get it merged :)